### PR TITLE
Enable strong mode toggling in dartpad web

### PR DESCRIPTION
**Pull request by Georgehe4**
#736
#759

This PR resolves feature requests for strong mode in dartpad.
1) Enable toggling of strong mode in dartpad. (by toggling strong=true)
2) Exported views are in strong mode if they were the embedded views were in strong mode

Example: http://crazy-monkey-721.appspot.com/c162b1c0eaa37142eeedf3226b47200d?strong=true

### CODE REVIEW

**Comment by chalin** [comment]
Will this also support: https://dartpad.dartlang.org?strong=true, so that we can open the base DartPad with strong mode enabled?

cc @kwalrath

**Comment by Georgehe4** [comment]
@chalin Yep this will add support to enable strong mode just like how you've linked. 
Embedded views that are in strong mode will open dartpad in strong mode as well.

File: lib/playground.dart [hunk 1]
```diff
@@ -193,6 +191,33 @@ class Playground implements GistContainer, GistController {
...
+   */
+  bool _parseStrongModeParam(String strongModeValueString) {
+    if (strongModeValueString == null) return false;
+    return (strongModeValueString == 'true' || strongModeValueString == 't');
+  }
+
+  /**
+   * Update query parameters for strong mode
+   */
+  void showStrong() {
```
Comment (lukechurch) [inline lib/playground.dart]: Consider another name such as 'setStrongModeFromUri'

File: lib/playground.dart [hunk 2]
```diff
@@ -193,6 +191,33 @@ class Playground implements GistContainer, GistController {
     _clearOutput();
   }
 
+
+  /**
+   * Return if strong mode should be enabled
```
Comment (lukechurch) [inline lib/playground.dart]: Return *true* if ...

**Comment by chalin** [comment]
When will this go live (is the update automatic)?

**Comment by Georgehe4** [comment]
#789

This will require a production push.

**Pull request merged**